### PR TITLE
Prevent multiple triggering

### PR DESCRIPTION
**Pull request by minsulee2**
Now only trigger functions once.

### CODE REVIEW

File: handlers/middleware.js [hunk 1]
```diff
@@ -141,6 +139,15 @@ class Middleware {
       throw ErrorUtil.setCustomError(422,
           `joi req validation error(${validationResult.error.details[0].message}).`);
     }
+  };
+
+  static preventMultipleTriggering = (req, res, next) => {
```
Comment (jiyoung-an) [inline handlers/middleware.js:144]: 👍

**Review by jiyoung-an — APPROVED** [review]
LGTM thank you for handle this error!

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Comment by minsulee2** [comment]
Thanks ! Time to merge!

**Pull request merged**